src/Mod/Draft/importSVG.py

```diff
@@ -244,36 +244,46 @@ def getcolor(color):
     (r, g, b, a)
         RGBA float tuple, where each value is between 0.0 and 1.0.
     """
+    if color == "none":
+        FreeCAD.Console.PrintMessage("Color defined to 'none', defaulting to black\n")
+        return (0.0, 0.0, 0.0, 1.0)
     if color[0] == "#":
-        # Color string '#12ab9f'
-        if len(color) == 7:
+        if len(color) == 7 or len(color) == 9: # Color string '#RRGGBB' or '#RRGGBBAA'
             r = float(int(color[1:3], 16) / 255.0)
```

Comment:
Actually `float` is not necessary here.

Reply:
I won't touch this now to keep uniformity